chainer/links/loss/hierarchical_softmax.py

```diff
@@ -11,6 +12,11 @@
 from chainer import variable
 
 
+LEAF = -1
```

Comment:
Can you write where you use these constants. `LEAF` is for `parent2child`, and `NOT_LEAF` is for `node2word`